frontend/public/components/utils/_label.scss

```diff
@@ -44,11 +69,19 @@ tags-input .host:active {
 }
 
 tags-input .tags {
+  align-items: flex-start;
```

Comment:
```suggestion
  align-content: flex-start;
```
`align-items` can be replaced with `align-content` which will prevent the extra vertical spacing.
<img width="714" alt="Screen Shot 2022-04-27 at 2 24 53 PM" src="https://user-images.githubusercontent.com/1874151/165594506-c79aeb00-0318-40d7-ac79-588c6c1b2a3f.png">
<img width="727" alt="Screen Shot 2022-04-27 at 2 25 13 PM" src="https://user-images.githubusercontent.com/1874151/165594537-632487ff-d9e1-4da3-afc0-61396af834de.png">